### PR TITLE
feat: Configure app-center navigation and pages for administration site - EXO-66546 - Meeds-io/MIPs#88

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+-->
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+          xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+    <page>
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml]: format code

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+  This file is part of the Meeds project (https://meeds.io/).
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml:4]: update header

File: app-center-webapps/src/main/resources/locale/navigation/portal/administration_en.properties [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#
+# This file is part of the Meeds project (https://meeds.io/).
```
Comment (azayati) [inline app-center-webapps/src/main/resources/locale/navigation/portal/administration_en.properties:2]: - update header
- you should declare the file into https://github.com/Meeds-io/app-center/blob/feature/experience/translations.properties#L22 like done for global site, in order to be considered by crowdin

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal-configuration.xml [hunk 4]
```diff
@@ -113,5 +113,43 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+      <name>new.portal.config.user.listener</name>
+      <set-method>initListener</set-method>
+      <type>org.exoplatform.portal.config.NewPortalConfigListener</type>
+      <description>this listener init the portal configuration</description>
+      <init-params>
+        <value-param>
+          <name>override</name>
+          <description>The flag parameter to decide if portal metadata is overriden on restarting server</description>
+          <value> ${exo.app-center.portalConfig.administration.metadata.override:true} </value>
+        </value-param>
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal-configuration.xml]:
```suggestion
```

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal-configuration.xml [hunk 5]
```diff
@@ -113,5 +113,43 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+              <string>portal</string>
+            </field>
+            <field name="templateLocation">
+              <string>war:/conf/app-center/portal</string>
+            </field>
+            <field name="override">
+              <boolean>${exo.app-center.portalConfig.administration.override:true}</boolean>
+            </field>
+            <field name="importMode">
+              <string>${exo.app-center.portalConfig.administration.importmode:insert}</string>
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal-configuration.xml]:
```suggestion
              <string>${exo.app-center.portalConfig.administration.importmode:merge}</string>
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+-->
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+          xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+  <page>
+    <name>appCenterAdminSetup</name>
+    <title>App center admin setup</title>
+    <access-permissions>*:/platform/administrators</access-permissions>
+    <edit-permission>manager:/platform/administrators</edit-permission>
+    <container template="system:/groovy/portal/webui/container/UIContainer.gtmpl" cssClass="singlePageApplication">
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/portal/administration/pages.xml]:
```suggestion
    <container id="singlePageApplicationContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl" cssClass="singlePageApplication">
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**